terasoluna-gfw-parent/pom.xml

```diff
@@ -392,28 +392,28 @@
     <!-- == TERASOLUNA == -->
     <terasoluna.gfw.version>5.5.0-SNAPSHOT</terasoluna.gfw.version>
     <!-- == Spring IO Platform == -->
-    <io.spring.platform.version>Brussels-SR5</io.spring.platform.version>
+    <io.spring.platform.version>Cairo-SR1</io.spring.platform.version>
     <!-- == MyBatis3 == -->
     <org.mybatis.version>3.4.5</org.mybatis.version>
     <org.mybatis.spring.version>1.3.1</org.mybatis.spring.version>
     <!-- == Dozer== -->
     <dozer.version>5.5.1</dozer.version>
     <!-- == Joda-Time == -->
     <joda-time.joda-time-jsptags.version>1.1.1</joda-time.joda-time-jsptags.version>
-    <jadira-usertype-core.version>5.0.0.GA</jadira-usertype-core.version>
+    <jadira-usertype-core.version>6.0.1.GA</jadira-usertype-core.version>
     <!-- == Logging == -->
     <org.lazyluke.version>0.2.7</org.lazyluke.version>
     <!-- == Project Properties == -->
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <archetype.test.skip>true</archetype.test.skip>
     <encoding>UTF-8</encoding>
-    <java-version>1.7</java-version>
+    <java-version>1.8</java-version>
     <!-- == Cargo Plugin == -->
     <cargo.jvmargs>-Xms512m -Xmx1024m</cargo.jvmargs>
     <cargo.daemon.url>http://localhost:18000</cargo.daemon.url>
-    <cargo.container.id>tomcat8x</cargo.container.id>
-    <cargo.container.url>http://archive.apache.org/dist/tomcat/tomcat-8/v${cargo.tomcat8.version}/bin/apache-tomcat-${cargo.tomcat8.version}.zip</cargo.container.url>
-    <cargo.tomcat8.version>8.5.20</cargo.tomcat8.version>
+    <cargo.container.id>tomcat9x</cargo.container.id>
+    <cargo.container.url>http://archive.apache.org/dist/tomcat/tomcat-9/v${cargo.tomcat9.version}/bin/apache-tomcat-${cargo.tomcat9.version}.zip</cargo.container.url>
+    <cargo.tomcat9.version>9.0.6</cargo.tomcat9.version>
   </properties>
   <profiles>
     <profile>
```

Comment:
In addition to what I reviewed out, there are a lot of warnings by the IDE.
We should create another Issue and respond.

Reply:
ok